torch/onnx/symbolic_opset17.py

```diff
@@ -54,3 +55,128 @@ def layer_norm(
         epsilon_f=eps,
         axis_i=axis,
     )
+
+
+@_onnx_symbolic("aten::stft")
+@symbolic_helper.parse_args("v", "i", "i", "i", "v", "b", "b", "b")
+@_beartype.beartype
+def stft(
+    g: jit_utils.GraphContext,
+    input: _C.Value,
+    n_fft: int,
+    hop_length: Optional[int] = None,
+    win_length: Optional[int] = None,
+    window: Optional[_C.Value] = None,
+    normalized: bool = False,
+    onesided: Optional[bool] = True,
+    return_complex: Optional[bool] = False,
```

Comment:
```suggestion
    onesided: bool = True,
    return_complex: bool = False,
```

can these values ever be None? If not:

Reply:
I was following the original [`torch.stft`](https://pytorch.org/docs/stable/_modules/torch/functional.html#stft) function. While I don't think `None` makes sense here, we should make the ONNX operator as compatible with the `torch.stft` function. Thoughts?

Reply:
Sound good. thanks for pointing this out!